public/app/features/dashboard/components/SaveDashboard/SaveDashboardDrawer.tsx

```diff
@@ -18,33 +17,24 @@ import { useDashboardSave } from './useDashboardSave';
 
 export const SaveDashboardDrawer = ({ dashboard, onDismiss, onSaveSuccess, isCopy }: SaveDashboardModalProps) => {
   const [options, setOptions] = useState<SaveDashboardOptions>({});
-
+  const previous = useSelector((store) => store.dashboard.getOriginal());
```

Comment:
Saving the original dashboard data to the store seems to work, but I'm not sure if there are any cases where it'll break.